packages/nc-gui/composables/useMultiSelect/cellRange.ts

```diff
@@ -0,0 +1,59 @@
+export type Cell =
+  | {
```

Comment:
Can be changed to 
```
export interface Cell {
  row: number | null
  col: number | null
}
```

Reply:
I actually meant here, that either both row and col can be null or none, that's not the same as what you wrote. But probably it's not very important and your variant is more readable.